redaxo/src/core/lib/error_handler.php

```diff
@@ -236,7 +236,7 @@ public static function handleError($errno, $errstr, $errfile, $errline)
 
         // silenced errors ("@" operator)
         if (0 === error_reporting()) {
-            return;
+            return false;
```

Comment:
Der error-handler muss bei silenced errors `false` liefern, damit es überhaupt von php weiter behandelt wird und in `error_get_last()` landet

Reply:
Fixed einen psalm error :)

https://github.com/vimeo/psalm/issues/2501